base/applications/rapps/lang/sq-AL.rc

```diff
@@ -96,7 +96,7 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "…", IDC_INSTALL_TEXT, 4, 5, 209, 35
     AUTORADIOBUTTON "&Instalo nga nje disk (CD or DVD)", IDC_CD_INSTALL, 10, 46, 197, 11, WS_GROUP
-    AUTORADIOBUTTON "Shkarko dhe instalo", IDC_DOWNLOAD_INSTALL, 10, 59, 197, 11, NOT WS_TABSTOP
+    AUTORADIOBUTTON "Shkarko &dhe instalo", IDC_DOWNLOAD_INSTALL, 10, 59, 197, 11, NOT WS_TABSTOP
```

Comment:
I would suggest `&Shkarko dhe instalo`

Reply:
I used &D because that is used also for en-US and does not collide.